lib/workspace.js

```diff
@@ -336,17 +337,53 @@ module.exports = {
     // local workspace files
     const questionBasePath = path.join(course_path, 'questions', question.qid);
     const localPath = path.join(questionBasePath, 'workspace');
+    const templatePath = path.join(questionBasePath, 'workspaceTemplates');
 
     // base workspace directory wherever we are uploading to
     const remoteDirName = `workspace-${workspace_id}-${workspace.version}`;
     const remotePath = path.join(remoteDirName, 'current');
 
-    // Check if we have any local files to upload
-    const localPathExists = await fse.pathExists(localPath);
+    const staticFiles = (
+      await async
+        .mapSeries(klaw(localPath), async (file) => {
+          return file.stats.isFile()
+            ? { name: path.relative(localPath, file.path), localPath: file.path }
+            : null;
+        })
+        .catch(() => {
+          // Path does not exist or is not accessible, do nothing
+          return [];
+        })
+    ).filter((file) => file);
+
+    const mustacheParams = { params: variant.params, correct_answers: variant.true_answer };
```

Comment:
Is there a specific use case you have in mind for making `correct_answers` available to the template?

Reply:
I mostly just added for completeness, but it could be useful for cases like a workspace-side preliminary check. I just copied the processing from the rubrics code, which adds these two plus the submission (which doesn't apply here). I don't mind removing it.

Reply:
I do like consistency here.

Reply:
I removed them in c8b8b6a but I can add them back. @mwest1066 what's your preference?